modules/postgres-circe/src/main/scala/doobie/postgres/circe/Instances.scala

```diff
@@ -34,6 +34,20 @@ object Instances {
       ).temap(a =>
         parse(a.getValue).leftMap(_.show))
 
+    implicit val arrayJsonGet: Get[Array[Json]] =
+      Get.Advanced.array[String](NonEmptyList.of("jsonb[]", "_jsonb"))
+        .map(_.map(jsonStr => parse(jsonStr).fold(throw _, identity)))
```

Comment:
I'd suggest using `Ge.temap` instead of `map` and thereby avoid `throw` inside `map` context.